src/include/kdb.h

```diff
@@ -1398,7 +1404,19 @@ typedef struct _kdb_vftabl {
      */
     void (*free_principal_e_data)(krb5_context kcontext, krb5_octet *e_data);
 
-    /* End of minor version 1 for major version 6. */
+    /* End of minor version 1. */
+
+    /*
+     * Optional: TODO
```

Comment:
What's this for?

Reply:
You mean the missing doc? I'll do asap, still thinking the API a bit.